src/current/cockroachcloud/connect-to-an-advanced-cluster.md

```diff
@@ -54,7 +54,8 @@ Private connectivity can be configured only after a cluster is created.
     - Set **Target** to **Published service**.
     - Set **Target service** to the value you copied from CockroachDB {{ site.data.products.cloud }} Console. If the endpoint's configured target service does not match, validation will fail.
     - Provide a value for **Endpoint name**. This is not used by CockroachDB {{ site.data.products.cloud }}.
-    - If it is not enabled, enable the Service Directory API, click **Enable global access**, and create a namespace in each region where your cluster is deployed.
+    - If it is not enabled, enable the Service Directory API, click **Enable global access**, and create a namespace in each region where your cluster is deployed. If the Service Directory API is not enabled, click Enable global access and create a namespace in each region where your cluster is deployed.
+When Service Directory and Global access are enabled, GCP automatically maps your private endpoint IPs to approved DNS names.CockroachDB Cloud requires DNS name in your connection info to match the approved pattern in your cluster’s certificates. With Service Directory and Global Access features enabled, you can simply replace `<ENTER-ENDPOINT-NAME>` in your connection information with the assigned name of the Endpoint name used in the above step.
```

Comment:
It looks like there's some redundancy here. Did you mean to remove the first sentence entirely?

Reply:
@jaylim-crl Could you PTAL, I've made a few small adjustments.